dataproc/create_cluster_test.go

```diff
@@ -52,7 +57,8 @@ func TestCreateCluster(t *testing.T) {
 	clusterName := fmt.Sprintf("go-cc-test-%s", tc.ProjectID)
 	region := "us-central1"
 
-	defer teardown(t, tc, clusterName, region)
+	deleteCluster(tc.ProjectID, clusterName, region) // Delete the cluster if it already exists.
```

Comment:
This might throw an error if it doesn't exist, but it seems like that's fine since we aren't capturing the error, if I understand correctly.

Reply:
Yes. Updated the comment.